docs/modules/about/pages/legal_notice.adoc

```diff
@@ -1,7 +1,7 @@
 [[welcome-legal-notice]]
 = Legal Notice
-{product-author}
-{product-version}
+{}
```

Comment:
I don't know how Antora works but is there a reason for this empty `{}` ?

Reply:
@RangerRick, my mistake. I did a global search to remove the "product-author" variable, and missed the {}.